test-java-lib-lombok/src/main/java/com/uber/lombok/LombokDTO.java

```diff
@@ -32,7 +32,9 @@
   "SameNameButDifferent" /* crashes with EP 2.6.0 */,
   "InlineMeInliner" /* crashes with EP 2.7.1 */
 })
-public class LombokBuilderInit {
+public class LombokDTO {
+  // Note: Field initialization is done by Lombok generated code. NullAway issues no error due to
+  // field being uninitialized.
```

Comment:
Not sure I follow this comment.  Why doesn't NullAway issue an error here?

Reply:
Lombok generates a constructor for this class, so there is actually a constructor initializing the field (which is called from the generated builder, but that's beside the point). That said, I don't know for a fact that NullAway checks the constructor (since it is `@Generated`) or if the mere presence of a constructor it can't analyze is enough to stop it from triggering an error here.

Reply:
Oh I get it! Basically it's because Lombok modifies this class to add a constructor. Cool, I think this is fine as is